benches/sync_semaphore.rs

```diff
@@ -28,14 +29,14 @@ async fn task(s: Arc<Semaphore>) {
     drop(permit);
 }
 
-fn uncontended_concurrent_multi(c: &mut Criterion) {
+fn uncontended_concurrent_multi(g: &mut BenchmarkGroup<WallTime>) {
     let rt = tokio::runtime::Builder::new_multi_thread()
```

Comment:
I found we call the following code many times：
```rust
   let rt = tokio::runtime::Builder::new_multi_thread()
        .worker_threads(6)
        .build()
        .unwrap();
```

```rust
 let rt = tokio::runtime::Builder::new_current_thread()
        .build()
        .unwrap();
```
Would it be better to write two constructor functions for them:

```rust
fn single_rt() -> Runtime{
    tokio::runtime::Builder::new_current_thread().build().unwrap()
}
```
and
```rust

fn multi_rt() -> Runtime{
    tokio::runtime::Builder::new_multi_thread() .worker_threads(6) .build() .unwrap()
}
```